### PR TITLE
Add list of all users in user profile. Configure routes

### PR DESCRIPTION
**Pull request by AlexSukov**

### CODE REVIEW

**Comment by AlexSukov** [comment]
@Magons it's based on #16, and rebased on **development**

File: app/assets/images/profile.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Magons) [inline app/assets/images/profile.svg]: Is this profile image?

File: app/views/users/show.html.haml [hunk 2]
```diff
@@ -1 +1,26 @@
...
+.row
+  .col-xs-12.col-sm-12.col-md-12.col-lg-12
+    %table.table.profile
+      %tr
+        %th{colspan: '2'}= @user.username
+      %tr
+        %td{colspan: '2'}= image_tag("profile.svg")
+      %tr
+        %td{colspan: '2'}= link_to 'Редактировать профиль', edit_user_registration_path, class: 'btn btn-link header-nav-link'
+      -if @user.comments!=[]
```
Comment (Magons) [inline app/views/users/show.html.haml]: `@user.comments != []`forgot spases

File: app/assets/images/profile.svg (hunk 1 above)
Comment (AlexSukov) [reply app/assets/images/profile.svg]: @Magons yes, just default image

**Comment by AlexSukov** [comment]
@Magons fixed

File: app/assets/images/profile.svg (hunk 1 above)
Comment (Magons) [reply app/assets/images/profile.svg]: why do you use this? if u can use glyphicon from bootstrap? or I'm wrong

**Comment by AlexSukov** [comment]
@Magons added user image like glyphicon. But I think it's not very correct, because glyphicons - is a text, not an image.

File: app/views/users/show.html.haml [hunk 3]
```diff
@@ -1 +1,27 @@
...
+  .col-xs-12.col-sm-12.col-md-12.col-lg-12
+    %table.table.profile
+      %tr
+        %th{colspan: '2'}= @user.username
+      %tr
+        %td{colspan: '2'}
+          %span.glyphicon.glyphicon-user.user
+      %tr
+        %td{colspan: '2'}= link_to 'Редактировать профиль', edit_user_registration_path, class: 'btn btn-link header-nav-link'
+      -if @user.comments != []
```
Comment (Magons) [inline app/views/users/show.html.haml]: `any?`

**Comment by AlexSukov** [comment]
@Magons, rebased and fixed

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -1,11 +1,9 @@
 class UsersController < ApplicationController
   before_action :authenticate_user!, only: [:new, :edit, :update, :destroy]
 
-  def index
-    @users = User.all
-  end
-
   def show
+    @users = User.all
     @user = User.find_by_username(params[:id])
```
Comment (Magons) [inline app/controllers/users_controller.rb:6]: why here `find_by_username`

File: app/views/users/show.html.haml [hunk 5]
```diff
@@ -19,3 +19,9 @@
       -else
```
Comment (Magons) [inline app/views/users/show.html.haml]: space

File: app/views/users/show.html.haml [hunk 6]
```diff
@@ -19,3 +19,9 @@
...
         %tr
           %td{colspan: '2'}= 'Вы не оставили ни одного комментария'
+      %tr
+        %td{colspan: '2'} Список всех пользователей:
+      -@users.each do |user|
```
Comment (Magons) [inline app/views/users/show.html.haml]: space

File: app/controllers/users_controller.rb (hunk 4 above)
Comment (AlexSukov) [reply app/controllers/users_controller.rb:6]: @Magons because it's device quide to do this)

**Comment by Magons** [comment]
@AlexSukov this PR has name "Add list of all users in user profile. Configure routes", but I don't see this. Can you explain what you do here?

**Comment by Magons** [comment]
@AlexSukov better call me

**Pull request merged**